types/stripe/index.d.ts

```diff
@@ -436,7 +437,7 @@ declare namespace Stripe {
              * dictionary containing a user’s credit card details (with the options shown
              * below). Stripe will automatically validate the card.
              */
-            external_account: string ;
+            external_account: string | IBankAccount;
```

Comment:
Maybe I'm missing something, but I think this is a different type.
`IBankAccount` is the type returned from `createExternalAccount`, while this object (`IExternalAccountCreationOptions`) is the object sent to this function, which has a different structure.
It seems like this type of object already exists, though not separated to its own interface in `IAccountUpdateOptions -> external_account`, so I would suggest extracting it and using it instead here.

Reply:
Oh that would make sense, I must have missed that but that does make sense because I realized there a disconnect between the create and update calls for some of the other methods as well. Like creating an account not all options are available in the CreationOptions interface for it so maybe there are more missing types than I thought. I will take a look and compare them against the docs and update this PR when I do.